orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -52,8 +52,8 @@ restore_db_from_local() {
 
   doLog "INFO Restoring $SERVICE"
 
-  psql --host="$HOST" --port="$PORT" --dbname="postgres" --username="$USERNAME" -c "drop database ${SERVICE};"
-  psql --host="$HOST" --port="$PORT" --dbname="postgres" --username="$USERNAME" -c "create database ${SERVICE};"
+  psql --host="$HOST" --port="$PORT" --dbname="postgres" --username="$USERNAME" -c "drop database ${SERVICE};" >>$LOGFILE 2>&1 || errorExit "Unable to drop database ${SERVICE}"
```

Comment:
Commenting here primarily to do a "request changes" review. I think that'll prevent the change from being merged in accidentally. See my general comment for the requested changes.